src/components/templates/Search/filterService.tsx

```diff
@@ -91,8 +148,31 @@ export default function FilterPrice({
           </Button>
         )
       })}
+      &nbsp;&nbsp;&nbsp;
```

Comment:
i think it would be better to handle this spacing using css rather than the HTML's non-breaking space